test/OpenFeature.Tests/OpenFeatureClientTests.cs

```diff
@@ -597,6 +597,20 @@ public async Task PassingABlankStringAsTrackingEventName_ThrowsArgumentException
         Assert.Throws<ArgumentException>(() => client.Track(" \n "));
     }
 
+    [Theory]
+    [InlineData(null)]
+    [InlineData("")]

```

Comment:
In case we should check for whitespace (https://github.com/open-feature/dotnet-sdk/pull/508/files#r2180899969)


```suggestion
    [InlineData("")]
    [InlineData(" ")]
```